### PR TITLE
chore: [1.1.x] use the product-version variables; simplify steps

### PR DESCRIPTION
**Pull request by nickboldt**
### What does this PR do?

chore: use the product-version variable in... (#99)

* chore: use the product-version variable in the download instructions; also provide a curl command to fetch the script to save users a step

Signed-off-by: Nick Boldt <nboldt@redhat.com>

* enable variable expansion in source block

Signed-off-by: Nick Boldt <nboldt@redhat.com>

* use product-bundle-version=1.1.0 in CSV/bundle references, and container tags :product-version instead of hardcoded 1.1

Signed-off-by: Nick Boldt <nboldt@redhat.com>

* dummy commit to test if gpg signing works for my commits

Signed-off-by: Nick Boldt <nboldt@redhat.com>

---------

Signed-off-by: Nick Boldt <nboldt@redhat.com>

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?
N/A (or see commit message above for issue number)

### How to test this PR?
N/A

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works, if one exists](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections What issues does this PR fix or reference and How to test this PR completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Comment by nickboldt** [comment]
NOTE: this looks like a reasonable change but fails in Pantheon so we need to figure out WHY before merging it. 

could be either related to adding a new variable to `artifacts/attributes.adoc` or it's a syntax issue with `[source,bash,subs="attributes"]` notation on code blocks (which asciidoctor was OK with, but I guess Pantheon is more crusty) :rofl:

**Review by rm3l — COMMENTED** [review]
While you're at it, could we also use this `product-version` variable in the "Deploying Red Hat Developer Hub in Elastic Kubernetes Services (EKS) using Helm Chart" section?
See https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/blob/main/admin/proc-rhdh-deploy-eks-using-helm.adoc?plain=1#L116-L119

**Comment by nickboldt** [comment]
> "using Helm Chart" section? 

https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/105/files#diff-d4e393f73a20f24734af71eac1fa0c6585a3dbf70d20d2b486852b0e761a18f2R116

> misleading

how so?

File: admin/proc-rhdh-deploy-eks-using-operator.adoc
Comment (rm3l) [inline admin/proc-rhdh-deploy-eks-using-operator.adoc:117]:
```suggestion
    rhdh-operator --patch '{"spec":{"template":{"spec":{"imagePullSecrets":[{"name":"rhdh-pull-secret"}]}}}}' \
```

While we're at it. I just re-tested this on EKS and the Deployment name needs to be updated.

**Comment by rm3l** [comment]
> > "using Helm Chart" section?
> 
> [#105 (files)](https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/105/files#diff-d4e393f73a20f24734af71eac1fa0c6585a3dbf70d20d2b486852b0e761a18f2R116)
> 
> > misleading
> 
> how so?

The previous message implied that 1.0.0-1 was the latest version of the Helm Chart, which is no longer accurate. But now, it is better with the right version (1.1.0).

**Review by rm3l — APPROVED** [review]

**Comment by hmanwani-rh** [comment]
@nickboldt Should we attempt to merge this and check if Pantheon accepts it this time? :)
Before that, we may need to rebase the branch.

**Comment by nickboldt** [comment]
Branch updated.

> Should we attempt to merge this and check if Pantheon accepts it this time?

Yes, as long as you're OK to keep publishing updates to  1.1 docs from the 1.1.x branch.

File: artifacts/attributes.adoc
Comment (nickboldt) [inline artifacts/attributes.adoc:5]:
```suggestion
:product-bundle-version: 1.1.1
```

File: artifacts/attributes.adoc
Comment (nickboldt) [inline artifacts/attributes.adoc:6]:
```suggestion
:product-chart-version: 1.1.1
```

File: artifacts/attributes.adoc
Comment (nickboldt) [inline artifacts/attributes.adoc:4]: should this move to 1.2 in main?

Comment (Gerry-Forde) [reply artifacts/attributes.adoc:4]: Yes for the main branch, it should be 1.2

Comment (nickboldt) [inline artifacts/attributes.adoc:4]:
```suggestion
:product-version: 1.2
```

**Comment by nickboldt** [comment]
> we may need to rebase the branch.

Done.

Let me know when you want to merge this... maybe after 1.1.1 is out?

**Review by linfraze — APPROVED** [review]
lgtm / I will build in Pantheon after the merge to see if any of the `[source]` changes break the build

File: admin/proc-rhdh-deploy-eks-using-operator.adoc
Comment (linfraze) [inline admin/proc-rhdh-deploy-eks-using-operator.adoc:290]: What is it that you are trying to achieve with this listing block? I'm not sure if this syntax is valid. Might be safer to revert to `[source]` since this change is not required and might break the build.

File: admin/proc-rhdh-deploy-eks-using-helm.adoc
Comment (nickboldt) [inline admin/proc-rhdh-deploy-eks-using-helm.adoc:112]:
```suggestion
[source,subs="attributes+"]
```

File: admin/proc-rhdh-deploy-eks-using-operator.adoc
Comment (nickboldt) [inline admin/proc-rhdh-deploy-eks-using-operator.adoc:81]:
```suggestion
[source,subs="attributes+"]
```

File: admin/proc-rhdh-deploy-eks-using-operator.adoc
Comment (nickboldt) [reply admin/proc-rhdh-deploy-eks-using-operator.adoc:290]: this is required to turn `{product-version}` into `1.2`

Comment (nickboldt) [reply admin/proc-rhdh-deploy-eks-using-operator.adoc:290]:
```suggestion
[source,subs="attributes+"]
```

File: admin/proc-using-aws-cognito-auth-provider.adoc
Comment (nickboldt) [inline admin/proc-using-aws-cognito-auth-provider.adoc:108]:
```suggestion
[source,subs="attributes+"]
```

File: installation/proc-install-rhdh-operator.adoc
Comment (nickboldt) [inline installation/proc-install-rhdh-operator.adoc:277]:
```suggestion
[source,subs="attributes+"]
```

**Comment by nickboldt** [comment]
> lgtm / I will build in Pantheon after the merge to see if any of the `[source]` changes break the build

See https://docs.asciidoctor.org/asciidoc/latest/verbatim/source-blocks/ for why I had `[source,console]` to define the markup.

See https://stackoverflow.com/questions/52528641/with-asciidoctor-how-to-pass-variables-in-source-and-example-blocks for how asciidoctor uses variable expansion in code blocks.

I don't know what Pantheon does with these instructions, however.

File: admin/proc-rhdh-monitoring-logging-aws.adoc
Comment (nickboldt) [inline admin/proc-rhdh-monitoring-logging-aws.adoc:45]:
```suggestion
[source,bash]
```

File: admin/proc-using-aws-cognito-auth-provider.adoc
Comment (nickboldt) [inline admin/proc-using-aws-cognito-auth-provider.adoc:209]:
```suggestion
[source,bash]
```

File: release-notes/con-release-notes-overview.adoc
Comment (nickboldt) [inline release-notes/con-release-notes-overview.adoc:44]:
```suggestion
[source,bash]
```

File: release-notes/con-release-notes-overview.adoc
Comment (nickboldt) [inline release-notes/con-release-notes-overview.adoc:55]:
```suggestion
  openshift-helm-charts/redhat-developer-hub --version {product-chart-version}
```

File: release-notes/con-release-notes-overview.adoc
Comment (nickboldt) [inline release-notes/con-release-notes-overview.adoc:52]:
```suggestion
[source,subs="attributes+"]
```

**Comment by nickboldt** [comment]
repalced by https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/171 and https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/172.

Will close this and submit a simpler PR to just test using attribute expansion in code blocks w/ Pantheon in the 1.1.x branch

**Comment by nickboldt** [comment]
new PR #175, new failure:

```
Transforming the AsciiDoc content to DocBook XML...
[31m[1mStartTag: invalid element name, line 797, column 25[0m
[31m[1mUnable to parse the AsciiDoc built DocBook XML[0m
[31m[1mAn error occurred when building the XML Feed. Please see the previous log messages for the cause.[0m
```

**Comment by nickboldt** [comment]
Success with:
* https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/177
* https://gitlab.cee.redhat.com/red-hat-developers-documentation/rhdh/-/commit/88cebf8eb4920ab157376de4eb70ee1f3d21a832
* https://dxp-docp-prod.apps.ext-waf.spoke.prod.us-west-2.aws.paas.redhat.com/documentation/en-us/red_hat_developer_hub/1.1/html-single/administration_guide_for_red_hat_developer_hub/index?lb_target=preview&check_logged_in=1#proc-rhdh-deploy-eks-using-helm_admin-rhdh

**Pull request closed without merging**
